packages/vc-handler-ld-local/README.md

```diff
@@ -0,0 +1,51 @@
+<!--suppress HtmlDeprecatedAttribute -->
+<h1 align="center">
+  <br>
+  <a href="https://www.sphereon.com"><img src="https://sphereon.com/content/themes/sphereon/assets/img/logo.svg" alt="Sphereon" width="400"></a>
+  <br>JSON-LD VC Handler Local (Typescript + React-Native) 
+  <br>
+</h1>
+
+---
+
+**Warning: This package still is in every early development. Breaking changes without notice will happen at this point!**
+
+---
+
+# vc-handler-ld-local
+
+A Veramo plugin to issue and verify JSON-LD based credentials and presentations using a react-native capable port of Digital Bazaars VC library.
+
```

Comment:
do we want some examples in here?

Reply:
Yes, I will add these separately, given I want to have a release out for Crenco ASAP. I created a comment in the ticket, so we can at least progress on this PR and create another one for the documentation update https://sphereon.atlassian.net/browse/MYC-144?focusedCommentId=34773